var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -85,6 +92,14 @@ def configure_args(self):
                  if '+dyninst' in self.spec else
                  ["--without-dyninst"])
 
+        args += (["--with-cuda=%s" % spec['cuda'].prefix]
+                 if '+cuda' in self.spec else
+                 ["--without-cuda"])
+
+        args += (["--with-cupti=%s" % spec['cupti'].prefix]
```

Comment:
This fails for me when testing because cputi is not an actual package spec. I think the the path should be `spec['cuda'].prefix.extras.CUPTI

Reply:
Hi @hyviquel, do you think you will have a chance to resolve this in the next few days?

Reply:
Done, I had to use `find_headers()` to make it more flexible because cupti is not in `extras/CUPTI` on my system.